tests/formatter_mock.go

```diff
@@ -3,7 +3,7 @@ package tests
 // DO NOT EDIT!
 // The code below was generated with http://github.com/gojuno/minimock (dev)
```

Comment:
@rekby can you please regenerate this mock file because the header has changed due to #34